syntax="proto3";

message TestUtilEmpty {
}

service TestUtilPing {
    rpc Ping(TestUtilEmpty) returns (TestUtilEmpty) {}
}
